packages/clack-prompts/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "@svelte-add/clack-prompts",
+    "version": "0.9.0",
```

Comment:
it might be nice to stick `"private": "true"` on these two new packages to prevent them from being published and have `@svelte-add/core` bundle them that way we don't have to pollute npm with what are hopefully short-lived temporary packages

Reply:
That sounds smart.

We should also add them here, otherwise the new packages will still be published I think. I'm not sure about this one, but at least that would explain why I added all those packages there initially:
https://github.com/svelte-add/svelte-add/blob/e7f414702ba319a649a22d4f73c4efc48d924ec6/package.json#L7

Reply:
heh, i actually haven't noticed the existence of that script until just now 😆

i don't think the publish script needs to be that complex. based on other projects i've setup changesets in, it should already respect the `"private": "true"` field, preventing the package from publishing

a simple `"ci:publish": "changesets publish"` script instead should do the job